### PR TITLE
Optimize paper-progress

### PR DESCRIPTION
**Pull request by blasten**
For: #16
- Before: 37 milliseconds avg creation time.
- After: 34 milliseconds avg creation time.

Not very significant, but It seems like the current cost is reasonable in terms of the overall complexity of paper-progress.
cc @sorvell

### CODE REVIEW

File: paper-progress.html [hunk 1]
```diff
@@ -278,17 +272,30 @@
...
+
+      this.setAttribute('aria-valuenow', value);
+      this.setAttribute('aria-valuemin', min);
+      this.setAttribute('aria-valuemax', max);
     },
 
-    _secondaryProgressChanged: function() {
-      this.secondaryProgress = this._clampValue(this.secondaryProgress);
-      this._setSecondaryRatio(this._calcRatio(this.secondaryProgress) * 100);
+    _hasSecondaryProgress: function(secondaryProgress) {
```
Comment (frankiefu) [inline paper-progress.html]: should be `secondaryRatio` in the argument.

**Comment by frankiefu** [comment]
The `indeterminate` animation seems busted now, e.g. it pauses for a second when it reaches the end and the expand/contract animations seem not right too.

**Comment by blasten** [comment]
you right man. I just fixed it, and now this elements has only 2 divs :)

**Comment by frankiefu** [comment]
LGTM, very nice! :+1:

**Pull request merged**